threatdownnebula.json

```diff
@@ -1,70 +1,50 @@
 {
     "appid": "5c742a84-46e6-4e28-8475-f26d9873216d",
-    "name": "Malwarebytes Cloud",
-    "description": "This app integrates with the Malwarebytes Cloud platform to perform prevention, detection, remediation, and forensics endpoint management tasks",
+    "name": "ThreatDown Nebula",
+    "description": "This app integrates with the ThreatDown (powered by Malwarebytes) Nebula platform to perform prevention, detection, remediation, and forensics endpoint management tasks",
     "type": "endpoint",
-    "product_vendor": "Malwarebytes",
-    "logo": "logo_malwarebytescloud.svg",
-    "logo_dark": "logo_malwarebytescloud_dark.svg",
+    "product_vendor": "ThreatDown",
+    "logo": "logo_threatdownnebula.svg",
+    "logo_dark": "logo_threatdownnebula_dark.svg",
     "product_name": "Malwarebytes Endpoint Protection",
```

Comment:
@mmomin-crest Yes, for now we would like to keep Malwarebytes Endpoint Protection as product_name, since it reflects company Endpoint Agent name compatible with the app. Despite name changed to ThreatDown, agent is still named as Malwarebytes Endpoint Protection.